setup.py

```diff
@@ -68,6 +69,8 @@
         "pillow>=6.1",
         "pyscal>=0.4.1",
         "scipy>=1.2",
+        "scikit-learn>=0.23.1",
+        "statsmodels>=0.12.1",
```

Comment:
Comment that `statsmodels` is in practice an indirect dependency through https://plotly.com/python/linear-fits/?
How much work would it be to not add `scikit-learn` as a new dependency, and do the [normalization](https://scikit-learn.org/stable/modules/generated/sklearn.preprocessing.MinMaxScaler.html) on our side?